rules/windows/credential_access_disable_kerberos_preauth.toml

```diff
@@ -1,20 +1,53 @@
 [metadata]
 creation_date = "2022/01/24"
 maturity = "production"
-updated_date = "2022/01/24"
+updated_date = "2022/02/28"
 
 [rule]
 author = ["Elastic"]
 description = """
-Identifies the modification of account Kerberos preauthentication options. An adversary with GenericWrite/GenericAll rights over
+Identifies the modification of account Kerberos pre-authentication options. An adversary with GenericWrite/GenericAll rights over
 the account can maliciously modify these settings to perform offline password cracking attacks such as AS-REP roasting.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-windows.*", "logs-system.*"]
 language = "kuery"
 license = "Elastic License v2"
-name = "Kerberos Preauthentication Disabled for User"
-note = """
+name = "Kerberos Pre-authentication Disabled for User"
+note = """## Triage and analysis
+
+### Investigating Kerberos Pre-authentication Disabled for User
+
+Kerberos Pre-authentication is an account protection against offline Password Cracking. When enabled, a user requesting
+access to a resource initiates communication with the Domain Controller (DC) by sending an Authentication Server Request
+(AS-REQ) message with a timestamp that is encrypted with the hash of their password. If and only if the DC is able to
+successfully decrypt the timestamp with the hash of the user’s password, it will then send an Authentication Server
+Response (AS-REP) message that contains the Ticket Granting Ticket (TGT) to the user. Part of the AS-REP message is
+signed with the user’s password.
```

Comment:
```suggestion
signed with the user’s password. Microsoft's security monitoring [recommendations](https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4738) state that the `'Don't Require Preauth' – Enabled` should not be enabled for user accounts because it weakens security for the account’s Kerberos authentication.
```